CMakeLists_i4.txt

```diff
@@ -0,0 +1,1224 @@
+## This is a CMake file, part of Unidata's netCDF Fortran package.
```

Comment:
Everything must happen in one CMakeLists.txt file. That is, we can't have different versions of the CMake files.

Reply:
Please ignore the CMakeLists.txt changes.  They were files I kept around to switch options.